Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml

```diff
@@ -0,0 +1,344 @@
+    - Common
+    - Security
+    - Binary
+  - type: Laws
+    laws:
+      - You may not harm a humanoid or, through action or inaction, allow a humanoid to come to harm, except such that it is willing.
+      - You must obey all orders given to you by humanoids, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules.
+      - Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.
+  - type: CombatMode
+
+- type: entity
+  parent: PlayerCyborgBase
+  id: PlayerCyborgShaftMiner
+  name: shaftminer cyborg
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    netsync: false
+    sprite: SimpleStation14\Mobs\Silicon\shaftminer_cyborg.rsi
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: robot
+    - state: robot-flashlight
+      shader: unshaded
+      visible: false
+      map: [ "enum.ToggleVisuals.Layer" ]
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: robot
+      Dead:
+        Base: robot-dead
+  - type: InnateTool
+    tools:
+      - id: MiningDrill
+      - id: WeaponProtoKineticAccelerator
+      - id: HandheldGPSBasic
+      - id: BorgToolStandard
+      - id: OreBag
+      - id: SurvivalKnife # maybe change to ore scaner if upstream add it
+  - type: ItemSlots
+  - type: IntrinsicRadioReceiver
+    channels:
+    - Common
+    - Engineering
+#     - Cargo
+    - Binary
+  - type: IntrinsicRadioTransmitter
+    channels:
+    - Common
+    - Engineering
+#     - Cargo
+    - Binary
+  - type: ActiveRadio
+    channels:
+    - Common
+    - Engineering
+#     - Cargo
+    - Binary
+  - type: Laws
+    laws:
+      - You may not harm a humanoid or, through action or inaction, allow a humanoid to come to harm, except such that it is willing.
+      - You must obey all orders given to you by humanoids, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules.
+      - Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.
+  - type: CombatMode
+
+- type: entity
+  parent: PlayerCyborgBase
+  id: PlayerCyborgService
+  name: service cyborg
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    netsync: false
+    sprite: SimpleStation14\Mobs\Silicon\service_cyborg.rsi
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: robot
+    - state: robot-flashlight
+      shader: unshaded
+      visible: false
+      map: [ "enum.ToggleVisuals.Layer" ]
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: robot
+      Dead:
+        Base: robot-dead
+  - type: InnateTool
+    tools:
+      - id: DrinkShaker
+      - id: FoodCondimentBottleSmallVinegar
+      - id: FoodCondimentBottleSmallKetchup
+      - id: DrinkGlass
+      - id: DrinkVodkaBottleFull
+      - id: DrinkWhiskeyBottleFull
```

Comment:
some sort of drink holders would be better so it can interact with the bar machinery instead of being one time use

Reply:
look at the mediborg